dev/com.ibm.ws.jaxws.2.3.clientcontainer/src/com/ibm/ws/jaxws23/client/security/LibertyJaxWsClientSSLOutInterceptor.java

```diff
@@ -0,0 +1,241 @@
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.client.security;
+
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.security.PrivilegedExceptionAction;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicReference;
+
+import javax.net.ssl.SSLSocketFactory;
+import javax.xml.namespace.QName;
+
+import org.apache.cxf.common.util.PropertyUtils;
+import org.apache.cxf.common.util.StringUtils;
+import org.apache.cxf.configuration.jsse.TLSClientParameters;
+import org.apache.cxf.interceptor.Fault;
+import org.apache.cxf.message.Message;
+import org.apache.cxf.phase.AbstractPhaseInterceptor;
+import org.apache.cxf.transport.Conduit;
+import org.apache.cxf.transport.http.HTTPConduit;
+import org.apache.cxf.transports.http.configuration.HTTPClientPolicy;
+import org.apache.cxf.ws.addressing.EndpointReferenceType;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.jaxws.metadata.ConfigProperties;
+import com.ibm.ws.jaxws.metadata.PortComponentRefInfo;
+import com.ibm.ws.jaxws.metadata.WebServiceRefInfo;
+import com.ibm.ws.jaxws.security.JaxWsSecurityConfigurationService;
+import com.ibm.wsspi.kernel.service.utils.AtomicServiceReference;
+
+import org.apache.cxf.Bus;
+import org.apache.cxf.interceptor.Fault;
+import org.apache.cxf.message.Message;
+import org.apache.cxf.phase.AbstractPhaseInterceptor;
+import org.apache.cxf.phase.Phase;
+import org.apache.cxf.service.model.EndpointInfo;
+import org.apache.cxf.transport.Conduit;
+import org.apache.cxf.transport.http.HTTPConduit;
+import org.apache.cxf.transport.http.HTTPConduitConfigurer;
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.cm.ManagedServiceFactory;
+
+import com.ibm.websphere.ras.ProtectedString;
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.websphere.ras.annotation.Sensitive;
+import com.ibm.ws.jaxws.JaxWsConstants;
+import com.ibm.ws.jaxws.metadata.ConfigProperties;
+import com.ibm.ws.jaxws.metadata.PortComponentRefInfo;
+import com.ibm.ws.jaxws.metadata.WebServiceRefInfo;
+import com.ibm.ws.jaxws.security.JaxWsSecurityConfigurationService;
+
+/**
+ * Used to set the SSL config on the Client side conduit. This removes the need to modify the
+ * HttpConduit directly through an extended LibertyHttpConduit 
+ */
+public class LibertyJaxWsClientSSLOutInterceptor extends AbstractPhaseInterceptor<Message> {
```

Comment:
In the common component we renamed this to be Security instead of SSL

Reply:
I hadn't intended this to remain when I pulled in my changes from the other PR, I will fix it.